rom/laghos.cpp

```diff
@@ -523,7 +523,7 @@ int main(int argc, char *argv[])
     if (fom_data)
     {
         const int num_tasks = mpi.WorldSize();
-        int unit;
+        int unit = 1;
```

Comment:
Wow, it was not initialized before? I wonder how this was even working in the past?

Reply:
This is a special case that is apparently not used in any of the parallel regression tests.